zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -44,6 +46,7 @@
 SHAKE = 18
 SINGLE = 1
 STATUS_TYPE_ATTR = 0x0055  # decimal = 85
+OPPLE_MFG_CODE = 0x115F
```

Comment:
The added `OPPLE_MFG_CODE` line seems to be unused.
If so, it can be removed.

Reply:
removed